test/box-luatest/gh_9923_invalid_iproto_index_name_mapping_on_the_server_side_test.lua

```diff
@@ -0,0 +1,88 @@
+local net = require('net.box')
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new({alias = 'master'})
+    cg.server:start()
+    cg.server:exec(function()
+        box.schema.create_space('test', {
+            id = 621,
+            temporary = true,
+            if_not_exists = true,
+            field_count = 3,
+            format = {
+                {name = "id", type = "unsigned"},
+                {name = "name", type = "string"},
+                {name = "arr1", type = "array"},
```

Comment:
Do you actually need this `arr1` field? Let's drop it to simplify the test.
Also, let's drop `field_count` and `temporary` attributes and explicit id of the space - they have nothing in common with bug we are fixing.

Reply:
fix